src/Application/Console/Formatters/Json.php

```diff
@@ -48,8 +48,10 @@ public function format(
                 'architecture' => $results->getStructure(),
                 'style' => $results->getStyle(),
                 'security issues' => $results->getTotalSecurityIssues(),
+                'fixed issues' => $results->getTotalFix(),
```

Comment:
remember to add this in the schema file also :)

Reply:
Done here : https://github.com/nunomaduro/phpinsights/pull/337/files#diff-76d5ffeac090c76ed5f4c581f02f3c53R71-R73

Is it correct ?